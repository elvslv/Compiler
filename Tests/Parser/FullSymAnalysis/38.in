var i, j: integer;
begin
	repeat
	until i > j;
end.