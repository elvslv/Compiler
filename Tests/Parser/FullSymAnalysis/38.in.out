var I : INTEGER;
var J : INTEGER;

begin        
|            
|___repeat   
|   |        
|   |___begin
|   |   |    
|   |   end  
|   |        
|   until    
|   |        
|   |___>    
|       |    
|       |___I
|       |    
|       |___J
|            
end          
